terraform/domains/environment_domains/config/production.tfvars.json

```diff
@@ -0,0 +1,23 @@
+{
+  "hosted_zone": {
+    "professional-development-for-teachers-leaders.education.gov.uk": {
+      "front_door_name": "s189p01-cpdinf-dom-fd",
+      "resource_group_name": "s189p01-cpdinf-dom-rg",
```

Comment:
Isn't the resource group name supposed to be the format s189p01-<shortname>domains-rg?

Reply:
Not now. Going forward they will change to this more standard format. We can reuse more code if we standardised the naming.